squid/datadog_checks/squid/squid.py

```diff
@@ -69,23 +69,25 @@
 
 
 class SquidCheck(AgentCheck):
+    HTTP_CONFIG_REMAPPER = {'cachemgr_user': {'name': 'username'}, 'cachemgr_passwd': {'name': 'password'}}
```

Comment:
isnt the old param `cachemgr_username` and `cachemgr_password`?

Reply:
yes! 😅